docs/csharp/language-reference/operators/boolean-logical-operators.md

```diff
@@ -0,0 +1,159 @@
+---
+title: "Boolean logical operators - C# Reference"
+description: "Learn about C# operators that perform logical negation, conjunction (AND), and inclusive and exclusive disjunction (OR) operations with Boolean operands."
+ms.date: 04/05/2019
+author: pkulikov
+f1_keywords: 
+  - "!_CSharpKeyword"
+  - "&_CSharpKeyword"
+  - "^_CSharpKeyword"
+  - "|_CSharpKeyword"
+  - "&&_CSharpKeyword"
+  - "||_CSharpKeyword"
+helpviewer_keywords: 
+  - "logical operators [C#]"
+  - "short-circuiting operators [C#]"
+  - "logical negation operator [C#]"
+  - "NOT operator [C#]"
+  - "! operator [C#]"
+  - "AND operator [C#]"
+  - "ampersand operator [C#]"
+  - "conjunction operator [C#]"
+  - "& operator [C#]"
+  - "exclusive OR operator [C#]"
+  - "XOR operator [C#]"
+  - "^ operator [C#]"
+  - "OR operator [C#]"
+  - "disjunction operator [C#]"
+  - "| operator [C#]"
+  - "conditional logical AND operator [C#]"
+  - "short-circuiting logical AND operator [C#]"
+  - "&& operator [C#]"
+  - "conditional logical OR operator [C#]"
+  - "short-circuiting logical OR operator [C#]"
+  - "|| operator [C#]"
+---
+# Boolean logical operators (C# Reference)
+
+The following operators perform logical operations with [bool](../keywords/bool.md) operands:
+
+- Unary [`!` (logical negation)](#logical-negation-operator-) operator.
+- Binary [`&` (logical AND)](#logical-and-operator-), [`|` (logical OR)](#logical-or-operator-), and [`^` (logical exclusive OR)](#logical-exclusive-or-operator-) operators. Those operators always evaluate both operands.
+- Binary [`&&` (conditional logical AND)](#conditional-logical-and-operator-) and [`||` (conditional logical OR)](#conditional-logical-or-operator-) operators. Those operators evaluate the second operand only if it's necessary.
```

Comment:
Given that listed operators are for `bool` operands, I haven't included `true` and `false` operators here; also, I assume their usage is very rare with comparison to the operators in this article. Anyway, the `true` and `false` operators are mentioned in the "Operator overloadability" section.

Please let me know, if it's fine.

Reply:
I agree. The `true` and `false` operators should be a separate topic.